src/InstrumentationEngine.Lib/InstrumentationMethod.cpp

```diff
@@ -89,15 +89,22 @@ HRESULT MicrosoftInstrumentationEngine::CInstrumentationMethod::InitializeCore(
 
     WCHAR wszModuleFullPath[MAX_PATH];
     memset(wszModuleFullPath, 0, MAX_PATH);
+#ifndef PLATFORM_UNIX
     if (!PathCanonicalize(wszModuleFullPath, m_bstrModuleFolder))
+#else
+    if (FAILED(PathCchAppend(wszModuleFullPath, MAX_PATH, m_bstrModuleFolder)))
```

Comment:
Shouldn't this be PathCchCanonicalize, so long as that is implemented in the PAL?

Reply:
Unfortunately that's not available in the PAL so I just reverted to what the code was before.

Reply:
That's not quite correct. Prior to the changes in #404, it initialized wszModuleFullPath with:

```cpp
wcscpy_s(wszModuleFullPath, MAX_PATH, m_bstrModuleFolder);
```

Reply:
I'm not certain why PathCanonicalize was introduced here in that PR.

Reply:
True, I can just revert the entire behavior; it's not really needed